### ISSUE TITLE
Add option to use connection string to add new DB connection

### ISSUE THREAD

**Issue by perryraskin** [slashbaseide/slashbase#53]
Would be great to have an option to use a connection string to add a database connection (at least for PostgreSQL).

**Comment by codezuma** [comment]
can i work on this @paraswaykole .

**Comment by paraswaykole** [comment]
Yes! You can. can you add it for all database types currently supported, MySQL,  PostgreSQL and MongoDB?

**Comment by codezuma** [comment]
@paraswaykole should implement it like that.

**Comment by paraswaykole** [comment]
Yes! That works.
